lighthouse-ci/README.md

```diff
@@ -0,0 +1,120 @@
+# Lighthouse CI Usage
+
+To monitor Lighthouse scores and prevent unexpected ad performance regressions, 
+you can integrate [Lighthouse CI (LHCI)](https://github.com/GoogleChrome/lighthouse-ci) 
+with your existing CI.
+
+>Lighthouse CI is a suite of tools that make continuously running, saving, 
+retrieving, and asserting against 
+[Lighthouse](https://github.com/GoogleChrome/lighthouse) results as easy as 
+possible.
+
+Today, this tool is compatible with GitHub Actions, Travis CI, Circle CI, GitLab CI, Jenkins, and Google Cloudbuild.
+
+If not familiar with LHCI, please familiarize yourself with their 
+[Getting Started documentation](https://github.com/GoogleChrome/lighthouse-ci/blob/master/docs/getting-started.md#github-app-method-recommended) 
+prior to starting.
+
+## Config
+
+LHCI natively supports plugins, including Publisher Ads Audits. To run
+Publisher Ads Audits in your LHCI reports, simply set the `settings.plugins` 
+attribute in the `lighthouserc.js` `collect` config.
+
+Example:
+```
+module.exports = {
+  ci: {
+    // ...
+    collect: {
+      settings: {
+        plugins: ['lighthouse-plugin-publisher-ads'],
+      },
+    },
+    // ...
+  }
+```
+
+Additionally, you must ensure that `lighthouse-plugins-publisher-ads` is 
+installed in your CI environment. The can be done by adding the line 
+`npm install -g lighthouse-plugin-publisher-ads@1.3.x` to your CI build rule. 
+
+We recommend setting a 
+specific major and minor versions (ex: 1.3.x) to ensure that unexpected regressions are not 
```

Comment:
Do you mean minor version (eg, v1.3) or patch version (v1.3.0)? I think patch version makes the most sense, since we definitely make scoring changes between minor versions.